jupyterlab/labbox_ephys_widgets_jp/package.json

```diff
@@ -31,7 +33,7 @@
   "scripts": {
     "build": "npm run build:lib && npm run build:nbextension",
     "build:labextension": "npm run clean:labextension && mkdirp labbox_ephys_widgets_jp/labextension && cd labbox_ephys_widgets_jp/labextension && npm pack ../..",
-    "build:lib": "tsc",
+    "build:lib": "tsc && npm run copy-files",
```

Comment:
Turns out that tsc only transpiles ts -> js and does not copy other files (such as .css) In order to handle css properly with the jupyterlab extension, we need to have a step that copies those css files into the lib/ directory. See the copy-files command below.